app/assets/stylesheets/components/projects/new_submission.scss

```diff
@@ -0,0 +1,16 @@
+.new-submission {
```

Comment:
I'm not sure if this belongs in its own component. It's very specific and seems like it will always be an element of the submission component. Whats your thoughts?